docs/ref/middleware.txt

```diff
@@ -332,6 +333,93 @@ in the :setting:`SECURE_REDIRECT_EXEMPT` setting.
 
 .. _nginx: https://nginx.org
 
+.. _referrer-policy:
+
+``Referrer-Policy``
```

Comment:
This shouldn't be rendered as the name of the header. Remove the backticks and hyphen.